package.json

```diff
@@ -27,23 +27,22 @@
   "main": "webpack.config.js",
   "keywords": [],
   "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@babel/preset-env": "^7.12.7",
+    "babel-loader": "^8.2.2",
     "eslint": "^7.15.0",
     "eslint-plugin-prettier": "^3.2.0",
     "nodemon": "^1.15.1",
     "prettier": "2.1.2",
-    "sass": "^1.26.10"
+    "sass": "^1.26.10",
+    "webpack": "^5.10.0",
+    "webpack-cli": "^4.2.0"
```

Comment:
Found the reason that these were installed as `dependencies`!

>Federalist runs `npm ci --production` which means: [...] Development dependencies (those listed under the “devDependencies” key in package.json) will not be installed

https://federalist.18f.gov/documentation/node-on-federalist/#installing-npm-dependencies

Reply:
I suspect this might also be why the `build-css` task uses `npx sass` rather than just bare `sass`, since `npx` will install a package if it's missing ([revised to require confirmation as of NPM 7.0.0](https://github.com/npm/cli/blob/latest/CHANGELOG.md#npx)). It seems like it'd be easier to install `sass` as a `dependencies`, and importantly more deterministic, since AFAIK there's no guarantee that the version of `sass` installed by `npx` would be the same as the one we define here in `devDependencies`..

Reply:
Updated to move Babel and Webpack back to `dependencies` in 54a0eab.